strawberry/ext/mypy_plugin.py

```diff
@@ -507,6 +520,7 @@ def collect_attributes(self) -> Optional[List[DataclassAttribute]]:
                 line=stmt.line,
                 column=stmt.column,
                 type=sym.type,
+                kw_only=False,
```

Comment:
Is this change related to the version bump?

Reply:
Unrelated to this PR, but why do we use the `dict` constructor here instead of using a `{}` literal, @patrick91?

Reply:
not 100% sure why this looks different than the mypy plugin: https://github.com/python/mypy/blob/master/mypy/plugins/dataclasses.py#L342

but if I did the change I used dict because I didn't want to change all the params to have quotes :D

Reply:
Yeah the mypy dataclasses plugin added support for `kw_only` in https://github.com/python/mypy/pull/10867, so we need it here as well.
I just put `False` here to see if things work. I'll compare our plugin to theirs and match the implementation.